pom.xml

```diff
@@ -664,6 +664,8 @@
         <mockito.version>5.2.0</mockito.version>
         <jacoco-maven-plugin.version>0.8.9</jacoco-maven-plugin.version>
         <javacrumbs.json-unit.version>2.38.0</javacrumbs.json-unit.version>
+
+        <surefire.rerunFailingTestsCount>2</surefire.rerunFailingTestsCount>
```

Comment:
Having this as a general property, I would not be sure this is the way.

Shouldn't we rather pass it as a property of the surefire maven plugin?

Reply:
actually your comment made me pause: do we want this behavior locally or only on CI ? in the latter case I would change the Jenkins file to pass the property to the command line